packages/mobile/src/harmony-native/components/Button/Button/Button.tsx

```diff
@@ -299,6 +300,7 @@ export const Button = (props: ButtonProps) => {
     <BaseButton
       onPressIn={handlePressIn}
       onPressOut={handlePressOut}
+      onLongPress={handlePressOut}
```

Comment:
Ok @dylanjeffers also tagging you for this part too :) 

Lmk if you have concerns -- PR description should cover it. It only improves the current UX, though we could definitely reconsider how long press works overall in our app